test/parser/libqasm/test_libqasm.py

```diff
@@ -78,3 +109,56 @@ def test_error(parser: Parser) -> None:
 def test_wrong_gate_argument_number_or_types(parser: Parser, error_message: str, circuit_string: str) -> None:
     with pytest.raises(IOError, match=error_message):
         parser.circuit_from_string(circuit_string)
+
+
+@pytest.mark.parametrize(
+    ("expected_output", "circuit_string"),
```

Comment:
In order to be consistent with other parametrized tests across the baseline, let's put `expected_output` as second element.